packages/bun-vscode/src/features/debug.ts

```diff
@@ -133,6 +148,35 @@ async function injectDebugTerminal(terminal: vscode.Terminal): Promise<void> {
   // disposing the terminal.
   setTimeout(() => terminal.dispose(), 100);
 }
+async function injectDebugTerminal2() {
+  const jsDebugExt = vscode.extensions.getExtension('ms-vscode.js-debug-nightly') || vscode.extensions.getExtension('ms-vscode.js-debug');
+  if (!jsDebugExt) return
+
+  await jsDebugExt.activate()
+  const jsDebug: import('@vscode/js-debug').IExports = jsDebugExt.exports;
+
+  return jsDebug.registerDebugTerminalOptionsProvider({
```

Comment:
`registerDebugTerminalOptionsProvider` is new right

Reply:
oh yeah it is

https://github.com/microsoft/vscode-js-debug/pull/2136/files#diff-3d8109446d747816948a48dd717a7477bd2ec4f72669d157fa07a700de7a3e0fR19